doc/languages-frameworks/r.section.md

```diff
@@ -94,6 +94,43 @@ Executing `nix-shell` will then drop you into an environment equivalent to the
 one above. If you need additional packages just add them to the list and
 re-enter the shell.
 
+## torch with GPU acceleration {#torch-gpu-accel}
+
+The `torch` package included in `nixpkgs` is based on the binary packages by the
+mlverse team. By default, the CPU version of the package is installed. To enable
+GPU acceleration, set `config.cudaSupport` to true when importing `nixpkgs` in a
+`shell.nix`. You will also need to use `nixglhost`. Here is an example of such a
+shell:
+
+```nix
+{ pkgs ? import <nixpkgs> {config.cudaSupport = true; }, lib ? pkgs.lib }:
+
+let
+ nixglhost-sources = pkgs.fetchFromGitHub {
+   owner = "numtide";
+   repo = "nix-gl-host";
+   rev = "main";
+   # Replace this with the hash Nix will complain about, TOFU style.
+   hash = "";
+ };
+
+ nixglhost = pkgs.callPackage "${nixglhost-sources}/default.nix" { };
+
+in pkgs.mkShell {
+  buildInputs = [
+    nixglhost
+    pkgs.rPackages.torch
+    pkgs.R
+  ];
+}
+
+```
+
+After executing `nix-shell` to drop in that environment, call `nixglhost R` to
```

Comment:
Again, `nixglhost` for non-NixOS systems.

Reply:
I have amended the docs to explain this is only for non-NixOS linux and that we are looking for people to fix torch for NixOS and darwin.